core/package.json

```diff
@@ -24,19 +24,20 @@
   "collection": "dist/collection/collection-manifest.json",
   "types": "dist/types/interface.d.ts",
   "files": [
-    "dist/",
+    "components/",
     "css/",
+    "dist/",
     "hydrate/",
     "loader/"
   ],
   "dependencies": {
-    "ionicons": "^5.1.2",
+    "@stencil/core": "^2.4.0",
+    "ionicons": "^5.5.0-1",
```

Comment:
Any chance there's a stable release of ionicons we can use?

Reply:
Yeah thinking we can release 5.5.0 once we know the dev build is good to go